README.md

```diff
@@ -85,10 +85,10 @@ And to the many of our beloved users who keep sending us feedback, you are an es
 
 ## Notes makes use of the following third-party libraries:
 
-* QMarkdownTextEdit
-* QSimpleUpdater
-* QAutostart
-* QX
+* [QMarkdownTextEdit](https://github.com/pbek/qmarkdowntextedit)
+* [QSimpleUpdater](https://github.com/alex-spataru/QSimpleUpdater)
+* [QAutostart](https://github.com/b00f/qautostart)
+* [Qxt]()
```

Comment:
Which repo are we using for Qxt?

Reply:
It appears to be [this one](https://bitbucket.org/libqxt/libqxt/src/master/), though we're only using some files.